src/flb_output.c

```diff
@@ -249,17 +249,6 @@ struct flb_output_instance *flb_output_new(struct flb_config *config,
         return NULL;
     }
 
```

Comment:
line 242: mask_id can be removed as the new implementation will use the id only.

Reply:
Good catch